app/controllers/admin_controller.rb

```diff
@@ -0,0 +1,25 @@
+class AdminController < ApplicationController
+  before_action :login_required
+  before_action :check_admin
+
+  def view_applications
+    @applicants = Admin.get_applicants
+  end
+
+  def review_applicant
```

Comment:
I think it's `edit` action because at the end there is a form that edits the status of the applicant. The applicant is either rejected or accepted